src/api/applicant/content-types/applicant/schema.json

```diff
@@ -66,6 +66,12 @@
       "type": "component",
       "repeatable": true,
       "component": "skills.skills"
+    },
+    "yearsExperience": {
+      "type": "decimal",
+      "required": true,
```

Comment:
Also note, in Strapi, required is validate on API level when the new records are created, not on database level. Thus existing `applicants` records will have this field empty, so you will need account for empty value in frontend.